### DISCUSSION TITLE
error "AttributeError: 'VariableElimination' object has no attribute 'variables'"

### DISCUSSION THREAD

**Discussion by w19539** [pgmpy/pgmpy#1440] [Q&A] [answered]
Hi There,

I am facing below issue when run "ordering = get_ordering(infer, variables=var, elimination_order=heuristic,
                        desc=f'Elimination order for {var} with no evidence computed through {heuristic} heuristic:')" in the 9th cell in the example at https://github.com/pgmpy/pgmpy_notebook/blob/master/notebooks/A%20Bayesian%20Network%20to%20model%20the%20influence%20of%20energy%20consumption%20on%20greenhouse%20gases%20in%20Italy.ipynb.

Could you help me ?

Thanks a lot!

~Catherine

**Comment by ankurankan** [comment, marked as answer]
@w19539 This was happening because of a bug (which got introduced in some recent changes). I have fixed it now, and if you install the latest dev branch from github, it should work. Thanks for pointing it out.

**Reply by w19539** [reply]
@ankurankan thanks for your quick reply. One more question, when will the fix be available in the official package? i.e. pip install pgmpy -upgrade

**Reply by ankurankan** [reply]
@w19539 We have a 3 month release cycle, so this should be released on 30th Sept.

**Reply by w19539** [reply]
Nice to know it. Thank you!

**Reply by NkosenhleMdluli** [reply]
Hi @ankurankan. I installed the latest dev branch from github using 'pip install git+https://github.com/pgmpy/pgmpy.git@dev' but I'm still getting the same error. Could you kindly assist?

**Reply by ankurankan** [reply]
@NkosenhleMdluli Could you please double-check that the dev version is getting used when you are running it? Because it works fine on my machine on dev.
